src/utils/permissions.js

```diff
@@ -0,0 +1,20 @@
+/* eslint-disable arrow-body-style */
+/* eslint-disable import/prefer-default-export */
+import { ROLE } from '../constants';
+
+export const hasDeleteArticlePermission = (user, authorId) => {
+  return user.role === ROLE.ADMIN || user._id === authorId.toString();
+};
```

Comment:
i think distracture like {role} = user , would be great here, but then overlapping with ROLE a bit, may be to {BASIC, ADMIN, EDITOR} = ROLE